csrc/device_lower/lower2device.cpp

```diff
@@ -382,7 +382,7 @@ void GpuLower::analysis(Fusion* fusion) {
   // functionality should be affected. New IterDomains may be created,
   // so it is expected that generated code may use diffrent variable
   // names
-  if (isOptionEnabled(EnableOption::IdModel)) {
+  if (true || isOptionEnabled(EnableOption::IdModel)) {
```

Comment:
Yes, I've been doing this with  all IdModle PRs, temporarily enabling it so that the build tests should in fact exercise this path. It'll be reverted back before merging.